.github/workflows/osx_release_lto.yml

```diff
@@ -32,14 +32,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  osx_11_aarch64:
+  osx_release_lto:
     # Run on push to the 'master' and release branches of tarantool/tarantool
     # or on pull request if the 'full-ci' label is set.
     if: github.repository == 'tarantool/tarantool' &&
         ( github.event_name != 'pull_request' ||
           contains(github.event.pull_request.labels.*.name, 'full-ci') )
 
-    runs-on: [ macos-11-self-hosted, aarch64 ]
+    runs-on: [ 'macos-${{ matrix.version }}-self-hosted', '${{ matrix.arch }}' ]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ '13' ]
```

Comment:
do you use a single-value matrix parameter for consistency between workflows?

Reply:
Yes. Also, we may decide to run this on macOS 12 in the future, and it will be very easy to do it.

Reply:
good